tests/tips_views_tests.py

```diff
@@ -52,13 +71,13 @@ def test_add_tip(self, generate_form):
         tip = form.save()
         assert tip in Tip.objects.all()
 
-    def test_edit_tip(self, generate_form, generate_tip):
+    def test_edit_tip(self, generate_form2, generate_tip):
```

Comment:
why not use generate_form instead of generate_form2?

Reply:
I think Or wanted to check that the data of the tip changes which is the purpose of this test.
generate tip and generate_form don't contain the same data, and he wanted to check that the data changed
Am I right @orIsrael ? 
I think generate_form could be good as well here, because they contain different authors.
I think you could change the content and use generate form (not 2) with different values, it would be more understandable in my opinion :)

Reply:
understand! thanks @LiorKGOW :)

Reply:
@orIsrael you can resolve this comment